components/style/gecko_bindings/sugar/style_complex_color.rs

```diff
@@ -17,6 +19,7 @@ impl From<nscolor> for StyleComplexColor {
 }
 
 impl StyleComplexColor {
+    /// Create a `StyleComplexColor` value that represents `currentColor`.
```

Comment:
[`currentcolor` is speced all-lowercased now](https://drafts.csswg.org/css-color-4/#currentcolor-color).